README.md

```diff
@@ -1,17 +1,74 @@
 # klaviyo-android-sdk
 
-## This project is still in pre-alpha
-## Breaking changes are still being made to the API. This is not yet intended for use
+## DISCLAIMER
+*This project is still in pre-alpha,
+breaking changes are still being made to the API. 
+This is not yet intended for public use*
+
+Android SDK allows developers to incorporate Klaviyo event and profile tracking functionality
+within native Android applications.
+The SDK assists in identifying users and tracking user events.
+Once integrated, your marketing team will be able to better understand your app users' needs and
+send them timely messages via FCM.
```

Comment:
Should this also mention about push notifications?

Reply:
yeaaa it should have the word push in there thanks

Reply:
copied that from the ios sdk its noah's fault :p